react/FilterNavigator.js

```diff
@@ -1,13 +1,13 @@
 import React, { Component, Fragment } from 'react'
 import PropTypes from 'prop-types'
-import { flatten, path, identity, contains, find, propEq, union, mergeAll, uniqBy, pick, keys, filter as filterRamda, props, map as mapRamda } from 'ramda'
-import ContentLoader from 'react-content-loader'
-import { withRuntimeContext, Link } from 'render'
-import SideBar from './components/SideBar'
 import { Button } from 'vtex.styleguide'
-import FilterIcon from './images/FilterIcon'
 import { FormattedMessage } from 'react-intl'
 import classNames from 'classnames'
+import { withRuntimeContext, Link } from 'render'
+import { flatten, path, identity, contains, find, propEq, union, mergeAll, uniqBy, pick, keys, filter as filterRamda, props, map as mapRamda } from 'ramda'
+import SideBar from './components/SideBar'
+import ContentLoader from 'react-content-loader'
```

Comment:
absolute imports first. also, move the relative ones to the same "block" as the other relative ones.